ufersa/plataforma-sabia
# Announcements

## Summary

Resolves #804 

https://www.loom.com/share/95e3e9254eec412bace0a69286757a47

## Relevant technical choices


## QA Steps

1. Access homepage
2. Click "Recursos > Banco de editais" in Header
3. Verify that you can interact with cards and buttons
4. Try to register a new announcement and validate all fields

## Checklist
